runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -919,4 +927,25 @@ private int ping(World world)
 		storedPings.put(world.getId(), ping);
 		return ping;
 	}
+
+	private Map<Integer, Boolean> getIsWestCoastWorld()
+	{
+		if (isWestCoastWorld == null)
+		{
+			// Map worlds to whether or not they are East/West coast
+			// https://abextm.github.io/cache2/#/viewer/enum/4992
+			// https://oldschool.runescape.wiki/w/Server
+			// As of writing this the runelite API doesn't give world coast information: https://api.runelite.net/runelite-1.10.18-SNAPSHOT/worlds.js
```

Comment:
We are RuneLite and so phrasing comments like this is confusing. If you want you can add it to the web service instead.

Reply:
Removed the comment to get rid of confusion.